docs/src/components/home/MuiStatistics.tsx

```diff
@@ -5,9 +5,9 @@ import Typography from '@mui/material/Typography';
 
 const data = [
   { title: '3.2M', metadata: 'Weekly downloads on npm' },
```

Comment:
```suggestion
  { title: '4M', metadata: 'Weekly downloads on npm' },
```

Based on https://npm-stat.com/charts.html?package=%40mui%2Fmaterial&package=%40material-ui%2Fcore&package=%40mui%2Fx-data-grid&package=%40mui%2Fx-date-pickers&from=2021-12-01&to=2022-12-01 :)